tests/unit-tests/__snapshots__/format.test.ts.snap

```diff
@@ -381,7 +383,7 @@ export interface Signature {
   module top level component. Explicit default export module top level
   component. Explicit default export module top level component. Explicit
   default export module top level component.
-</template> as TemplateOnlyComponent<Signature>
+</template> as TemplateOnlyComponent<Signature>;
```

Comment:
according to prettier and having <template> expressed as an functionDeclaration AST node this should have a semi
@gitKrystan ?

Reply:
Seems fine.